'make clean' always failed

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

'make clean' are always failed on master branch.
201911 branch has no problem.

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1. `make init`
2. `make configure PLATFORM=broadcom` (`PLATFORM=vs` is also affected)
3. `make clean`

**Describe the results you received:**
```
make: *** No rule to make target 'target/debs/buster/redis-tools_5.0.3-3~bpo9+2_amd64.deb-clean', needed by 'target/debs/buster/redis-tools-dbgsym_5.0.3-3~bpo9+2_amd64.deb-clean'.  Stop.
make[1]: *** [Makefile.work:224: clean] Error 2
```

**Describe the results you expected:**

no error.

**Additional information you deem important (e.g. issue happens only occasionally):**

found at commit 20698aa57b

--- a/rules/redis.mk
+++ b/rules/redis.mk
@@ -1,30 +1,30 @@
 # redis package
-
-REDIS_VERSION = 5.0.3-3~bpo9+2
-
-REDIS_TOOLS = redis-tools_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
-$(REDIS_TOOLS)_SRC_PATH = $(SRC_PATH)/redis
-$(REDIS_TOOLS)_DEPENDS += $(LIBHIREDIS_DEV)
-$(REDIS_TOOLS)_RDEPENDS += $(LIBHIREDIS)
 # TODO: docker-sonic-p4 depends on redis-tools in Jessie.
 #       Remove this file and src/redis after that resolved.
 ifneq ($(BLDENV),buster)
-SONIC_MAKE_DEBS += $(REDIS_TOOLS)
+
+    REDIS_VERSION = 5.0.3-3~bpo9+2
+
+    REDIS_TOOLS = redis-tools_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
+    $(REDIS_TOOLS)_SRC_PATH = $(SRC_PATH)/redis
+    $(REDIS_TOOLS)_DEPENDS += $(LIBHIREDIS_DEV)
+    $(REDIS_TOOLS)_RDEPENDS += $(LIBHIREDIS)
+    SONIC_MAKE_DEBS += $(REDIS_TOOLS)
+
+    REDIS_TOOLS_DBG = redis-tools-dbgsym_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
+    $(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_TOOLS_DBG)))
+
+    REDIS_SERVER = redis-server_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
+    $(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_SERVER)))
+
+    REDIS_SENTINEL = redis-sentinel_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
+    $(REDIS_SENTINEL)_DEPENDS += $(REDIS_SERVER)
+    $(REDIS_SENTINEL)_RDEPENDS += $(REDIS_SERVER)
+    $(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_SENTINEL)))
+
+    # The .c, .cpp, .h & .hpp files under src/{$DBG_SRC_ARCHIVE list}
+    # are archived into debug one image to facilitate debugging.
+    #
+    DBG_SRC_ARCHIVE += redis
+
 endif
-
-REDIS_TOOLS_DBG = redis-tools-dbgsym_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
-$(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_TOOLS_DBG)))
-
-REDIS_SERVER = redis-server_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
-$(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_SERVER)))
-
-REDIS_SENTINEL = redis-sentinel_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
-$(REDIS_SENTINEL)_DEPENDS += $(REDIS_SERVER)
-$(REDIS_SENTINEL)_RDEPENDS += $(REDIS_SERVER)
-$(eval $(call add_derived_package,$(REDIS_TOOLS),$(REDIS_SENTINEL)))
-
-# The .c, .cpp, .h & .hpp files under src/{$DBG_SRC_ARCHIVE list}
-# are archived into debug one image to facilitate debugging.
-#
-DBG_SRC_ARCHIVE += redis
-

